apps/etterlatte-brev-api/src/main/kotlin/no/nav/etterlatte/rivers/DistribuerBrev.kt

```diff
@@ -22,14 +19,15 @@ import org.slf4j.LoggerFactory
 
 internal class DistribuerBrev(
     private val rapidsConnection: RapidsConnection,
+    private val vedtaksbrevService: BrevService,
     private val distribusjonService: DistribusjonService
 ) : River.PacketListener {
     private val logger = LoggerFactory.getLogger(DistribuerBrev::class.java)
 
     init {
         River(rapidsConnection).apply {
             eventName(BrevEventTypes.JOURNALFOERT.toString())
-            validate { it.requireKey("brevId", "journalpostId", "distribusjonType", "mottakerAdresse") }
```

Comment:
Hvorfor kan ikke adressen her ha den dataen som ligger på brevet?

Reply:
Fordi det allerede er et mangelfult adresseobjekt på river. Pluss at jeg uansett synes det er bedre å hente rett fra basen, i stedet for å sende store objekter rundt på riveren.

Reply:
Det skal jeg ikke si noe på!